std_msgs/Header header

float32 roll	# roll angle [deg]
float32 pitch	# pitch angle [deg]
byte mode 	# control mode {0/position, 1/velocity}
